packaging/tools/post.sh

```diff
@@ -627,9 +627,16 @@ function install_app() {
   fi
 }
 
-function install_TDengine() {
-    echo -e "${GREEN}Start to install TDengine...${NC}"
-    log_print "start to install TDengine"
+function checkDirectory() {
+    if [ ! -d "/usr/local/bin" ]; then
+      sudo mkdir -p /usr/local/bin
```

Comment:
need consider an environment like docker has no sudo command.

Reply:
Adapted to different platforms